src/mapeo-project.js

```diff
@@ -619,9 +619,10 @@ export class MapeoProject extends TypedEmitter {
       ])
 
     await Promise.all(
-      namespacesWithoutAuth.map((namespace) =>
-        this.#coreManager.deleteData(namespace, { deleteOwn: true })
-      )
+      namespacesWithoutAuth.flatMap((namespace) => [
+        this.#coreManager.getWriterCore(namespace).core.close(),
```

Comment:
what's the context for this?

Reply:
ah wait is it related to this note in your PR description?

> (but still close your writer Hypercore).

Reply:
Yeah, that's exactly it. (Let me know if that's wrong!)

Reply:
i think that makes sense

Reply:
I'm not 100% sure it's the right call. My goal is:

- When you leave, we want to close all Hypercores.
- When you leave, we want to delete all Hypercore data except yours.

Does this express that correctly? (And is my goal correct?)